dev-haskell/pipes-group/files/pipes-group-1.0.12-import.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/src/Pipes/Group.hs b/src/Pipes/Group.hs
+index df928d7..95a4128 100644
+--- a/src/Pipes/Group.hs
++++ b/src/Pipes/Group.hs
+@@ -16,6 +16,9 @@ type Joiner         a m x = Groups a m x -> 'Producer' a m x
+ -}
+ 
+ {-# LANGUAGE RankNTypes #-}
++{-# LANGUAGE PackageImports #-}
++--‘Control.Monad.Trans.Free’ is present is two packages:
++--free-5.1.2, transformers-free-1.0.1
+ 
+ module Pipes.Group (
+     -- * Lenses
+@@ -48,8 +51,8 @@ module Pipes.Group (
+     ) where
+ 
+ import Control.Monad.Trans.Class (lift)
+-import Control.Monad.Trans.Free (FreeF(Pure, Free), FreeT(FreeT, runFreeT))
+-import qualified Control.Monad.Trans.Free as F
++import "free" Control.Monad.Trans.Free (FreeF(Pure, Free), FreeT(FreeT, runFreeT))
++import qualified "free" Control.Monad.Trans.Free as F
```

Comment:
Such patches are fine if upstream already accepted them. Otherwise it's painful to maintain on every revbump. I would just disable tests and enable them in next release.

Reply:
Ok, I'm dropping it, and delegate this to a discussion with upstream if there is appetite on their side. I thought there was an interest to fix broken testsuites, but it makes sense to limit patches to the basic cases "won't build/work". After all, whenever I see a patch present in a bump candidate, I got some serious head scratching to do to sort out if it's still relevant (and when it's unclear, it can get nasty!).